### PR TITLE
FOGL-2395: test modified to make compatible with rpi0

### PR DESCRIPTION
**Pull request by Singhal-Vaibhav**

### CODE REVIEW

File: tests/system/python/rpi/test_e2e_rpi_ephat.py [hunk 1]
```diff
@@ -69,21 +69,23 @@ def get_statistics_map(self, foglamp_url):
...
+                          remove_data_file, skip_verify_north_interface, remove_directories, enable_schedule,
+                          foglamp_url, start_north_pi_server_c, pi_host, pi_port, pi_token, wait_time):
         """ This fixture clones given south & filter plugin repo, and starts south and PI north C instance
 
         """
 
         add_south(SOUTH_PLUGIN, south_branch, foglamp_url, service_name=SVC_NAME)
 
-        start_north_pi_server_c(foglamp_url, pi_host, pi_port, pi_token, taskname=TASK_NAME, start_task=False)
+        if not skip_verify_north_interface:
```
Comment (praveen-garg) [inline tests/system/python/rpi/test_e2e_rpi_ephat.py]: I think we should find some way to send data to north interface for these (another is modbus rtu) special tests /cc @ashwinscale

File: tests/system/python/rpi/test_e2e_rpi_ephat.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
 TASK_NAME = "North v2 PI"
 
 
-@pytest.mark.skipif('raspberrypi' != os.uname()[1], reason="RPi only (ePhat) test")
+@pytest.mark.skipif('raspberrypi' != os.uname()[1] and 'raspizero' != os.uname()[1], reason="RPi only (ePhat) test")
```
Comment (praveen-garg) [inline tests/system/python/rpi/test_e2e_rpi_ephat.py]: or

Comment (praveen-garg) [reply tests/system/python/rpi/test_e2e_rpi_ephat.py]: better to say ` os.uname()[1] in ('raspberrypi', 'raspizero')`

**Comment by Singhal-Vaibhav** [comment]
Leftover stuff will be done in FOGL-3037

**Pull request merged**